text/entities/0264-resource-and-entities.md

```diff
@@ -0,0 +1,284 @@
+| identifying_attributes_keys | repeated string | Attribute Keys that identify the entity.<br/>MUST not change during the lifetime of the entity. The Id must contain at least one attribute.<br/><br/>These keys MUST exists in Resource.attributes.<br/><br/>Follows OpenTelemetry common attribute definition. SHOULD follow OpenTelemetry semantic conventions for attributes.| now a reference |
+| descriptive_attributes_keys | repeated string | Descriptive (non-identifying) attribute keys of the entity.<br/>MAY change over the lifetime of the entity. MAY be empty. These attribute keys are not part of entity's identity.<br/><br/>These keys MUST exist in Resource.attributes.<br/><br/>Follows any value definition in the OpenTelemetry spec - it can be a scalar value, byte array, an array or map of values. Arbitrary deep nesting of values for arrays and maps is allowed.<br/><br/>SHOULD follow OpenTelemetry semantic conventions for attributes.| now a reference |
+
+## How this proposal solves the problems that motivated it
+
+Let's look at some motivating problems from the [Entities Proposal](https://docs.google.com/document/d/1VUdBRInLEhO_0ABAoiLEssB1CQO_IcD5zDnaMEha42w/edit#heading=h.atg5m85uw9w8):
+
+### Problem 1: Commingling of Entities
+
+We embrace the need for commingling entities in Resource and allow downstream users to interact with the individual entities rather than erasing these details.
+
+### Problem 2: Lack of Precise Identity
+
+Identity is now clearly delineated from description via the Entity portion of Resource. When Entity is used for Resource, only identifying attributes need to be interacted with to create resource identity.
+
+### Problem 3: Lack of Mutable Attributes
+
+This proposal offers two solutions going forward to this:
+
+- Descriptive attributes may be mutated without violating Resource identity
+- Entities whose lifetimes do not match SDK may be attached/removed from Resource.
+
+### Problem 4: Metric Cardinality Problem
+
+Via solution to (2) we can leverage an identity synthesized from identifying attributes on Entity.  By directly modeling entity lifetimes, we guarantee that identity changes in Resource ONLY occur when source of telemetry changes.  This solves unintended metric cardinality problems (while leaving those that are necessary to deal with, e.g. collecting metrics from phones or browser instances where intrinsic cardinality is high).
+
+## Entity WG Rubric
+
+The Entities WG came up with a rubric to evaluate solutions based on shared
+beliefs and goals for the overall effort.  Let's look at how each item is
+achieved:
+
+### Resource detectors (soon to be entity detectors) need to be composable / disjoint
+
+Entity detection and Resource Manager now fulfill this need.
+
+### New entities added by extension should not break existing code
```

Comment:
What does "extension" refer to?

Reply:
I create a new semantic convention, entity detector, etc.  The existence of this new "type" should not cause breakages throughout the ecosystem.